elements/pfe-jump-links/custom-elements-manifest.config.js

```diff
@@ -1,5 +1,5 @@
 import { pfeCustomElementsManifestConfig } from '@patternfly/pfe-tools/custom-elements-manifest.js';
 
 export default pfeCustomElementsManifestConfig({
-  globs: ['pfe-*.ts'],
+  globs: ['pfe-*.ts', 'Base*.ts'],
```

Comment:
We could add this as a patch note in the changeset.

Reply:
since i was able to put everything in the controller, we don't actually export a base class.

composition > inheritance :muscle: